examples/cmake/setup_examples.cmake

```diff
@@ -108,6 +108,8 @@ function (add_examples examples_paths)
             set(CPU_ARCHITECTURE "intel_intel64")
         elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
             set(CPU_ARCHITECTURE "arm_aarch64")
+        elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv64")
+            set(CPU_ARCHITECTURE "riscv64_riscv64")
```

Comment:
Nit: Should it be "riscv_rv64" following the scheme of "arm_aarche64"?

Reply:
Yeah, I had thought about that. It's just naming, and I'm not sure why the repeated values are here, but I'm copying the style. Ideally, I would like to see just a single architecture string (and then maybe have x86 rather than Intel). I think that would be another pull request, as the changes then are not associated with just RISC-V